ibft/ibft.go

```diff
@@ -61,10 +66,15 @@ type ibftImpl struct {
 	instanceConfig      *proto.InstanceConfig
 	ValidatorShare      *storage.Share
 	Identifier          []byte
-	leaderSelector      leader.Selector
 
 	// flags
 	initFinished bool
+
+	// channels
+	CurrentInstanceResultChan chan *InstanceResult
```

Comment:
i think we can remove the current from the name as it a chan

Reply:
+1